jazz_environment-event-handler/test/test.js

```diff
@@ -0,0 +1,237 @@
+// =========================================================================
+// Copyright © 2017 T-Mobile USA, Inc.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// =========================================================================
+
+const chai = require('chai');
+const assert = require('chai').assert;
+const expect = require('chai').expect;
+const should = require('chai').should();
+const chaiAsPromised = require('chai-as-promised'); chai.use(chaiAsPromised);
+const request = require('request');
+const AWS = require('aws-sdk-mock');
+const awsContext = require('aws-lambda-mock-context');
+const async = require("async");
+const sinon = require('sinon')
+const rp = require('request-promise-native');
+const index = require('../index');
+const logger = require('../components/logger');
+const config = require('../components/config');
+const fs = require('fs');
+
+var event, context, configData, tokenResponseObj, kinesisPayload;
+
+describe('jazz environment handler tests: ', function () {
+	var sandbox;
+	beforeEach(function () {
+		sandbox = sinon.sandbox.create();
+
+		context = awsContext();
+		context.functionName = context.functionName + "-test"
+		var payloadFormat = fs.readFileSync('test/KINESIS_PAYLOAD.json');
+		kinesisPayload = JSON.parse(payloadFormat);
+		tokenResponseObj200 = {
+			"statusCode": 200,
+			"body": {
+				"data": {
+					"user_id": "JazzAdmin",
+					"name": "Jazz Admin",
+					"email": "jazz@serverless.com",
```

Comment:
because serverless.com is a real domain name, I would replace it with something else.